third_party/move/move-compiler-v2/tests/checking-lang-v2.4/cross_module_package_access_success.move

```diff
@@ -18,6 +18,8 @@ module 0xc0ffee::m {
         s: S,
```

Comment:
We should have some systematic testing of the generated file format in `third_party/move/move-compiler-v2/tests/file-format-generator`. Potentially you can reuse some of the transactional tests. But effectively, you probably just need to iterate the different forms of struct/enum types we can have.